osclib/accept_command.py

```diff
@@ -140,8 +194,32 @@ def accept_other_new(self):
             self.create_new_links(self.api.project, req['packages'][0], oldspecs)
```

Comment:
`oldspecs` here is another example of the need for that method, also note it does not use `expand` is that correct in this case?

Reply:
it's correct.